cpp/basix/finite-element-utils.cpp

```diff
@@ -0,0 +1,668 @@
+// Copyright (c) 2020 Chris Richardson & Matthew Scroggs
+// FEniCS Project
+// SPDX-License-Identifier:    MIT
+
+#include "finite-element-utils.h"
+#include "dof-transformations.h"
+#include "e-brezzi-douglas-marini.h"
+#include "e-bubble.h"
+#include "e-crouzeix-raviart.h"
+#include "e-hermite.h"
+#include "e-hhj.h"
+#include "e-lagrange.h"
+#include "e-nce-rtc.h"
+#include "e-nedelec.h"
+#include "e-raviart-thomas.h"
+#include "e-regge.h"
+#include "e-serendipity.h"
+#include "math.h"
+#include "polyset.h"
+#include <basix/version.h>
+#include <cmath>
+#include <concepts>
+#include <limits>
+#include <numeric>
+#define str_macro(X) #X
+#define str(X) str_macro(X)
+
+using namespace basix;
+
+template <typename T, std::size_t d>
```

Comment:
Maybe use 
```cpp
template <typename T, std::size_t d>
using mdspan_t = MDSPAN_IMPL_STANDARD_NAMESPACE::mdspan<
    T, MDSPAN_IMPL_STANDARD_NAMESPACE::dextents<std::size_t, d>>;
```
here so this code won't be affected by changes to impl::mdspan_t (not even sure where this is defined). Also put inside and anonymous namespace.